pal_event_server/lib/src/tesp_server.dart

```diff
@@ -37,10 +39,13 @@ import 'daemon/daemon.dart' as daemon;
 import 'pal_server/pal_commands.dart' as pal_commands;
 import 'sqlite_database/sqlite_database.dart';
 import 'allowlist.dart';
+import 'allowlist_default_rules.dart';
+
+final _logger = Logger('Tesp_Logger');
```

Comment:
nit: to be consistent with other logger names, maybe use `TespServer`

Reply:
done